### PR TITLE
docs: README

### LINKED ISSUE #989: [Docs]: update README.md
### Current State

**Description of the current state/issue**:

Related: https://github.com/apache/apisix-website/pull/984, https://github.com/apache/apisix-website/pull/972

After above PRs are merged, We need to update README.md to help contributors develop and build.

**Link to the page**:
none

**Screenshots (if relevant)**:
none

### Desired State

**Description of the change you are proposing**:

### Would you like to fix this issue?

Yes! Count me in

### PR DESCRIPTION
**Pull request by SkyeYoung**
Fixes: #989

Changes:

refactor README

Screenshots of the change:

### CODE REVIEW

**Comment by SkyeYoung** [comment]
Please check https://github.com/SkyeYoung/apisix-website/tree/refactor-readme#apache-apisix-website to have a preview.

File: README.md [hunk 1]
```diff
@@ -1,20 +1,105 @@
 
-# Apache APISIX® Website
+<h1 style="position: relative" align="center">Apache APISIX® Website</h1>
 
-[<img align="right" width="150" src="./website/static/img/join-slack.png">](https://apisix.apache.org/docs/general/join)
+<div align="center">
+The website of <a href="https://github.com/apache/apisix/">Apache APISIX®</a>
 
-The website of [Apache APISIX®](https://apisix.apache.org/), a cloud-native microservices api gateway.
+A cloud-native microservices api gateway
```
Comment (juzhiyuan) [inline README.md]:
```suggestion
A cloud-native microservices API Gateway
```

File: README.md [hunk 2]
```diff
@@ -1,20 +1,105 @@
...
+<a href="https://apisix.apache.org/docs/general/join"><img  width="150" src="./website/static/img/join-slack.png"></a>
+</div>
 
 ## Usage
 
+> If you want to write a blog or fix some blog-related issues, please read [Apache APISIX Blog Contributing Guide](http://apisix.apache.org/docs/general/blog) first.
+
+### Preparation
+
+First, you need to install `node.js`, `yarn`, `git`.
```
Comment (juzhiyuan) [inline README.md]:
```suggestion
First, you need to install `Node.js`, `Yarn`, `GIt`.
```

File: README.md [hunk 3]
```diff
@@ -1,20 +1,105 @@
...
+    │   ├── pages
+    │   ├── shaders
+    │   └── theme # swizzled any Docs/Blog component
+    ├── static
+    └── docusaurus.config.js
+```
+
+## Acknowledge
+
+Special thanks to [docusaurus](https://docusaurus.io/).
```
Comment (juzhiyuan) [inline README.md]:
```suggestion
Special thanks to [Docusaurus](https://docusaurus.io/).
```

**Review by juzhiyuan — DISMISSED** [review]

**Comment by juzhiyuan** [comment]
Hello, please resolve those errors :) https://github.com/apache/apisix-website/runs/6323027779?check_suite_focus=true

**Comment by SkyeYoung** [comment]
> Hello, please resolve those errors :) [apache/apisix-website/runs/6323027779?check_suite_focus=true](https://github.com/apache/apisix-website/runs/6323027779?check_suite_focus=true)

Fixed

**Review by juzhiyuan — APPROVED** [review]

**Review by SylviaBABY — APPROVED** [review]

**Pull request merged**